config/__init__.py

```diff
@@ -56,6 +56,18 @@ def __init__(self) -> None:
         if not found:
             invalid = True
 
+        self.slack_client_id, found = self.validate_env_var("SLACK_CLIENT_ID")
+        if not found:
+            invalid = True
+
+        self.slack_client_secret, found = self.validate_env_var("SLACK_CLIENT_SECRET")
+        if not found:
+            invalid = True
+
+        self.slack_scopes, found = self.parse_env_list("SLACK_SCOPES")
+        if not found:
+            invalid = True
+

```

Comment:
I'm curious. Why do we still need this if the Slack config is added through the DTO and page of the CMS? Wouldn't this be quite redundant?

Reply:
I agree; I think this will be refactored in the next sprint.